tests/expectations/metrics/test_core.py

```diff
@@ -3922,7 +3922,7 @@ def test_median_metric_spark(spark_session):
     assert results == {desired_metric.id: 2}
 
 
-def test_distinct_metric_spark(spark_session):
+def test_value_counts_metric_spark(spark_session):
```

Comment:
Could you mark these tests with `unit`, `integration`, or `e2e`?

Reply:
Added `integration` mark ✅